src/hotspot/os/posix/os_posix.cpp

```diff
@@ -89,7 +89,7 @@
 
 static jlong initial_time_count = 0;
 
-static int clock_tics_per_sec = 100;
+static int64_t clock_tics_per_sec = 100;
```

Comment:
why do we need to do this? It would get promoted in any expression using 64-bit variables

Reply:
src/hotspot/os/posix/os_posix.cpp:1211:31: warning: conversion from 'long int' to 'int' may change value [-Wconversion]
 1211 |   clock_tics_per_sec = sysconf(_SC_CLK_TCK);
      |                        ~~~~~~~^~~~~~~~~~~~~

Reply:
Although I did the opposite for the linux version (ie casted the result of sysconf instead).

Reply:
I'm really starting to think we should just be using `long` in platform specific code, given it acts the same as `intx` on this platform.